nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -122,6 +123,7 @@ def __init__(
         xscaling=True,
         untie_biases=True,
         conv_kernel_size=31,
+        conv_norm_type="batch_norm",
```

Comment:
For consistency with jasperblock, maybe just use `batch` - `layer` etc

Reply:
dropped the feature.